.golangci.yml

```diff
@@ -147,6 +147,37 @@ linters-settings:
           - pkg: github.com/gravitational/teleport/lib/web$
             desc: 'lib/web should not be imported to prevent increasing binary size'
         list-mode: lax
+      # Prevent importing dependencies that require CGO into tools that are
+      # meant to be built with CGO_ENABLED=0
+      cgo:
+        files:
+          # Tests can do anything
+          - "!$test"
+          - '**/tool/tbot/**'
+          - '**/lib/client/**'
+        deny:
+          - pkg: github.com/gravitational/teleport/lib/devicetrust/authn$
+            desc: '"devicetrust/authn" requires CGO on darwin'
+          - pkg: github.com/gravitational/teleport/lib/devicetrust/enroll
+            desc: '"devicetrust/enroll" requires CGO on darwin'
+          - pkg: github.com/gravitational/teleport/lib/devicetrust/native
+            desc: '"devicetrust/native" requires CGO on darwin'
+          - pkg: github.com/gravitational/teleport/lib/lib/bpf
+            desc: '"lib/bpf" requires CGO'
+          - pkg: github.com/gravitational/teleport/lib/vnet/daemon
+            desc: '"vnet/daemon" requires CGO'
+          - pkg: github.com/gravitational/teleport/lib/system/signal
+            desc: '"lib/system/signal" requires CGO'
+          - pkg: github.com/gravitational/teleport/lib/inventory/metadata
+            desc: '"lib/inventory/metadata" requires CGO'
+          - pkg: github.com/gravitational/teleport/lib/desktop/rdp/rdpclient
+            desc: '"lib/desktop/rdp/rdpclient" requires CGO'
+          - pkg: github.com/gravitational/teleport/lib/pam
+            desc: '"lib/pam" requires CGO'
+          - pkg: github.com/gravitational/teleport/lib/srv/uacc
+            desc: '"lib/srv/uacc" requires CGO'
+          - pkg: github.com/gravitational/teleport/lib/cgroup
```

Comment:
If you wanna add a few more:

- lib/auth/touchid -> CGO on Darwin (behind `touchid` build tag)
- lib/auth/webauthnwin -> CGO on Windows (no build tag!)
- lib/auth/webauthncli -> imports both packages above

Reply:
Nevermind this, lib/client uses those.